gobblin-core/build.gradle

```diff
@@ -57,6 +57,7 @@ dependencies {
   compile externalDependency.hadoopHdfs
 
   runtime externalDependency.protobuf
+  runtime externalDependency.hadoopAdl
```

Comment:
With the comment below regarding service provider, I think it's better to add a module for ADL. Will remove this line and add a module for ADL.